routes/chat.js

```diff
@@ -19,6 +21,12 @@ router.get('/channels/:channelId', auth.isAuthAnonim, chatController.getChannel)
 router.post('/init-chat', auth.isAuth, chatController.initChat);
 router.post('/init-chat-anonymous', auth.isAuthAnonim, chatController.initChatAnonymous);
 router.post('/users/:targetUserId', auth.isAuth, chatController.getMyAnonProfile);
+router.post(
+  '/channels/read',
+  BodyValidationMiddleware.setSignedChannelAsRead,
+  auth.isAuthV2,
+  setSignedChannelAsRead
+);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new endpoint `/channels/read` is using `auth.isAuthV2` for authentication. If this middleware function is designed to handle both authenticated and anonymous users, then it's fine. However, if it only handles authenticated users, you might want to consider adding a separate middleware function to handle anonymous users as well, similar to how it's done in the `/init-chat-anonymous` route. This would ensure that anonymous users can also mark channels as read.
```diff
  router.post(
    '/channels/read',
    BodyValidationMiddleware.setSignedChannelAsRead,
-   auth.isAuthV2,
+   [auth.isAuthV2, auth.isAuthAnonim],
    setSignedChannelAsRead
  );
```
Please disregard this comment if `auth.isAuthV2` already handles anonymous users.


<!-- This is an auto-generated comment by OSS CodeRabbit -->